src/guides/hyperlinks.md

```diff
@@ -2,14 +2,16 @@
 A hyperlink is clickable-text. When the user clicks on the text, it directs them to a certain URL.
 
 ## General Hyperlinks
-You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md) and in [webhook](../guides/webhooks.md) content/description.
+You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md), [webhook](../guides/webhooks.md) content/description, [slash command](../guides/slashCommands.md) response content and ephemeral [$onInteraction](../callbacks/onInteraction.md) response content.
```

Comment:
Ok, like, no `ephemeral`. All `$onInteraction[]` response content allows for hyperlinks, not only ephemeral.
And two more minor things, you put `,` before `and`, and change `$onIntraction` to look like the other functions.

Reply:
> Ok, like, no ephemeral. All $onInteraction[] response content allows for hyperlinks, not only ephemeral.

https://user-images.githubusercontent.com/70456337/178481618-8e9885a2-e6e4-4962-9718-c5da03dd5045.mp4